views/api/helpers.js

```diff
@@ -0,0 +1,10 @@
+export const handleApiResponse = async (response) => {
+  if (response.status !== 200 && response.status !== 204)
```

Comment:
```suggestion
  if (!response.ok)
```

https://developer.mozilla.org/en-US/docs/Web/API/Response/ok